toga/setup.py

```diff
@@ -9,6 +9,7 @@
         # Automatically installed platform backends
         ':sys_platform=="win32"': ["toga-winforms==%s" % version],
         ':sys_platform=="linux"': ["toga-gtk==%s" % version],
+        ':"freebsd" in sys_platform': ["toga-gtk==%s" % version],
```

Comment:
Does this actually work? Like - have you tested it? I didn't think the `sys_platform` syntax was "freeform Python". Can you provide a reference to docs that says this works, and/or another package that does this?

Reply:
Yes, it works. See [this](https://github.com/python-poetry/poetry/issues/2138#issuecomment-747130500) for example.